baselines/commoncontrols.md

```diff
@@ -1238,3 +1239,131 @@ To configure additional services per the policy:
+  - [T1213: Data from Information Repositories](https://attack.mitre.org/techniques/T1213/)
+
+#### GWS.COMMONCONTROLS.18.4v0.3
+
+The action for the custom DLP policy SHOULD be set to block external sharing.
+
+- _Rationale:_ Data Loss Prevention (DLP) rules help identify and limit the sharing of sensitive content, protecting agency information. Blocking and/or having warnings on these DLP-scanned files from being shared with users can reduce the risk of unintentional introduction of sensitive content.
+- _Last modified:_ October 23, 2024
+
+- MITRE ATT&CK TTP Mapping
+  - [T1530: Data from Cloud Storage](https://attack.mitre.org/techniques/T1530/)
+  - [T1048: Exfiltration Over Alternative Protocol](https://attack.mitre.org/techniques/T1048/)
+    - [T1048:002: Exfiltration Over Alternative Protocol: Exfiltration Over Asymmetric Encrypted Non-C2 Protocol](https://attack.mitre.org/techniques/T1048/002/)
+  - [T1213: Data from Information Repositories](https://attack.mitre.org/techniques/T1213/)
+
+### Resources
+- [GWS Admin Help \| Protect sensitive information using DLP](https://support.google.com/a/topic/7556687?hl=en&ref_topic=7558840&fl=1&sjid=4459086914710819343-NA)
+- [GWS Admin Help \| Use Workspace DLP to prevent data loss](https://support.google.com/a/answer/9646351?hl=en&visit_id=638635679011849528-69139467&ref_topic=9646660&rd=1)
+- [GWS Admin Help \| Create DLP for Drive rules and custom content detectors](https://support.google.com/a/answer/9655387)
+- [GWS Admin Help \| Prevent data leaks from Chat messages & attachments](https://support.google.com/a/answer/10846568)
+- [GWS Admin Help \| Prevent data leaks in email & attachments](https://support.google.com/a/answer/14767988?fl=1&sjid=4620103790740920406-NA)
+
+### Prerequisites
+- None
+
+### Implementation
+
+#### GWS COMMONCONTROLS 18 Common Instructions
+1.  Sign in to the [Google Admin Console](https://admin.google.com).
+2.  Select **Menu -\> Security -\> Access and data control -\> Data protection**.
+3.  Under Data protection rules and detectors click **Manage Rules**. 
+4.  For each DLP rule necessary to meet the standards of an agency:
+5.  Click **Add rule** -\> **New rule** or click **Add rule** -\> **New rule from template**. For templates, select a template from the Templates page.
+
+#### GWS.COMMONCONTROLS.18.1v0.3 Instructions
+1. For new rules:
+2. In the **Name** section, add the name and description of the rule.
+3. In the **Scope** section, apply this rule to the entire domain and click **Continue**.
+4. In the **Apps** section, choose the trigger for **Google Drive, Drive files** then click **Continue**.
+5. In the **Conditions** section, click **Add Condition**.
+    1. Select **All content** then **Matches predefined data type** then **Global - Credit card number** then **very low** and enter 1 for **Minimum unique matches** **minimum match count**.
```

Comment:
A few things,
- If you remove "Add Conditions" from above, be sure to add it here
- Maybe spell out the field names instead of assuming the order?
- Since the baseline does not require or recommend a specific threshold, maybe leave that open here.

What do think about this:
```suggestion
    1. Click **Action Condition**. For **Content type to scan** select **All content**. For **What to scan for** select **Matches predefined data type**. For **Select data type** select **Global - Credit card number**. Select the remaining condition properties according to agency need.
```

I won't spell it out, but if you like those suggestions, be sure to add that for the other conditions.

Reply:
@mdueltgen My bad, my suggestion had a typo. Instead of "Action Condition" it should say "Add Condition." Since it's my typo, I'm can fix it if you'd like.